src/components/navigation/searchBar/SearchBar.styles.js

```diff
@@ -6,11 +6,10 @@ top: 0;
 left: 0;
```

Comment:
should we keep the search bar even though it's not functioning?

Reply:
Good question! @ThorbenBender ?